### ISSUE TITLE
Get continuous location info: DEVICE_PRECISE_LOCATION 

### ISSUE THREAD

**Issue by phaniram-sayapaneni** [actions-on-google/actions-on-google-nodejs#379]
Hello,

Is it possible to get continuous lat, long info for upto 30 minutes, with just 1 permission request.
Using DEVICE_PRECISE_LOCATION. In the documentation, it says it would ask for permission. But how to get data continuously with out,  asking permission multiple times. 

This is to develop actions to enable customers get order delivered as soon as the person reaches the locations. Any help please?

**Comment by phaniram-sayapaneni** [comment]
@Fleker  need help. Thanks!

**Comment by Fleker** [comment]
No, you need to request permission each time you get the user location.

**Comment by phaniram-sayapaneni** [comment]
But, I want to track the user journey, what if the user is stuck in traffic or had an emergency, this is very important for my application. I only need to track for 30 mins. Can you set a single permission to send location continuously every 1 min for next 30 mins?

**Comment by Fleker** [comment]
No, you need to request permission each time.
